modules/integration/tests-integration/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSTransactionInboundTestCase.java

```diff
@@ -0,0 +1,169 @@
+/*
+*  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.esb.jms.inbound.transport.test;
+
+import org.apache.axiom.om.OMElement;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.extensions.servers.jmsserver.client.JMSQueueMessageConsumer;
+import org.wso2.carbon.automation.extensions.servers.jmsserver.client.JMSQueueMessageProducer;
+import org.wso2.carbon.automation.extensions.servers.jmsserver.controller.config.JMSBrokerConfigurationProvider;
+import org.wso2.carbon.integration.common.admin.client.LogViewerClient;
+import org.wso2.carbon.integration.common.utils.mgt.ServerConfigurationManager;
+import org.wso2.carbon.logging.view.stub.types.carbon.LogEvent;
+import org.wso2.esb.integration.common.clients.inbound.endpoint.InboundAdminClient;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import org.wso2.esb.integration.common.utils.JMSEndpointManager;
+import org.wso2.esb.integration.common.utils.servers.ActiveMQServer;
+
+import javax.jms.JMSException;
+
+/**
+ * class tests checking for a successful/unsuccessful jms transactions
+ */
+public class JMSTransactionInboundTestCase extends ESBIntegrationTest {
+
+	private LogViewerClient logViewerClient = null;
+	private ActiveMQServer activeMQServer = new ActiveMQServer();
+	private LogEvent[] logs;
+	String message = null;
+
+	@BeforeClass(alwaysRun = true)
+	protected void init() throws Exception {
+
+		activeMQServer.startJMSBrokerAndConfigureESB();
+		super.init();
+
+		ServerConfigurationManager serverConfigurationManager = new ServerConfigurationManager(
+				new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN));
+		OMElement synapse =
+				esbUtils.loadResource(
+						"/artifacts/ESB/jms/inbound/transport/jms_inbound_transaction.xml");
+
+		updateESBConfiguration(JMSEndpointManager.setConfigurations(synapse));
+		message = String.valueOf(
+				esbUtils.loadResource("artifacts/ESB/jms/inbound/transport/message.xml"));
+		InboundAdminClient inboundAdminClient =
```

Comment:
if you're using the addInboundEndpoiint method in ESBIntegrationTest you don't need to initialize this right?